frontend/src/components/UI/HeroDescription/HeroDescription.jsx

```diff
@@ -23,7 +23,8 @@ const HeroDescription = ({ label, description, urls }) => {
 
     const [textStyle, setTextStyle] = useState(styles.hidden);
     const [buttonStyle, setButtonStyle] = useState(styles.visible);
-
+    // TODO: Once the endpoint is ready, set this to true
```

Comment:
or just remove the conditional altogether ?

Reply:
that would work as well